packages/yara/yara.0.1/opam

```diff
@@ -0,0 +1,26 @@
+opam-version: "1.2"
+maintainer: "Anton Kochkov <anton.kochkov@gmail.com>"
+authors: "Anton Kochkov <anton.kochkov@gmail.com>"
+homepage: "https://github.com/XVilka/yara-ocaml"
+bug-reports: "https://github.com/XVilka/yara-ocaml/issues"
+license: "MIT"
+dev-repo: "https://github.com/XVilka/yara-ocaml.git"
+build: [
+  ["jbuilder" "subst" "-n" name] {pinned}
+  ["jbuilder" "build" "-p" name "-j" jobs]
+]
+depends: [
+  "jbuilder" {build}
+  "ppx_deriving" {build}
+  "core"
+  "ctypes"
+  "ctypes-foreign"
+]
+depexts: [
+  [["archlinux"] ["yara"]]
+  [["debian"] ["libyara3"]]
```

Comment:
I'm pretty sure you need ```libyara-dev``` instead here. And it's probably kind of the same thing for the other distributions